src/signals/incident/containers/IncidentContainer/saga.js

```diff
@@ -66,15 +65,16 @@ export function* getClassification(action) {
 
 export function* getQuestionsSaga(action) {
   const incident = yield select(makeSelectIncidentContainer)
-  const { category, subcategory } =
-    action.type === UPDATE_INCIDENT
-      ? action.payload
-      : getIncidentClassification(incident, action.payload)
+  const { category, subcategory } = getIncidentClassification(
+    incident,
+    action.payload
+  )
   if (
     !configuration.featureFlags.fetchQuestionsFromBackend ||
     !category ||
     !subcategory
   ) {
+    yield put(getQuestionsError())
```

Comment:
I get that the state needs to be reset, but dispatching the `getQuestionsError` when the feature flag `fetchQuestionsFromBackend` is 'false' doesn't seem right; an error didn't occur, functionality just isn't used. Dispatching an error action doesn't give a developer the right feedback in the Redux dev console. Also, semantically it's not the correct thing to do.
I would suggest dispatching something like `setLoadingData()`.